contracts/BasePoolConfig.sol

```diff
@@ -79,22 +79,22 @@ contract BasePoolConfig is Ownable, Initializable {
     // The ERC20 token this pool manages
     IERC20 public underlyingToken;
 
-    // Evaluation Agents (EA) are the risk underwriting agents that associated with the pool.
+    // Evaluation Agents (EA) are the risk underwriting agents that are associated with the pool.
     address public evaluationAgent;
 
     uint256 public evaluationAgentId;
 
     PoolConfig internal _poolConfig;
 
-    AccruedIncome internal _accuredIncome;
+    AccruedIncome internal _accruedIncome;
```

Comment:
I'm seeing these names in HTML files too. Should I update those directly or through some sort of file generation tool?

Reply:
html is generated. At some point, it'll be updated.

Reply:
So I should remove all these?

Reply:
No, I meant typos in HTML files would be fixed if we merge this pr and regenerate them. As @wisdant is the primary owner of the contracts, he will need to review and sign off this PR.